packages/libs/eda/src/lib/core/hooks/workspace.ts

```diff
@@ -88,8 +111,18 @@ export function useEntityAndVariable(descriptor?: VariableDescriptor) {
   );
 }
 
-export function useCollectionVariables(entity: StudyEntity) {
-  return useMemo(() => findCollections(entity).flat(), [entity]);
+export function useEntityAndVariableCollection(
+  descriptor?: VariableCollectionDescriptor
+) {
+  const entities = useStudyEntities();
+  return useMemo(
+    () => descriptor && findEntityAndVariableCollection(entities, descriptor),
+    [descriptor, entities]
+  );
+}
+
+export function useVariableCollections(entity: StudyEntity) {
+  return useMemo(() => findVariableCollections(entity).flat(), [entity]);
```

Comment:
```suggestion
  return useMemo(() => findVariableCollections(entity), [entity]);
```

haven't tested it in action, but the TS return type is the same and I couldn't see any deeper structure returned from `findVariableCollections`